docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -129,6 +129,8 @@ Testing
   in NSRDB (:issue:`733`)
 * Added tests for methods in bifacial.py.
 * Added tests for changes to cell temperature models.
+* Add tests configuration for bare python environment (no conda).
```

Comment:
I can't comment on the test configuration being added. But here: past tense "Added" or present tense "Add"?  No reason to hold up the PR.

Reply:
There's a mix of past and present throughout the file, so I'm going to keep it as is for now. I'm ok setting a loose standard if someone wants to do the background research into what the rest of the python ecosystem uses. I can see arguments for either past or present. Separate issue.

Reply:
[Python](https://docs.python.org/3/whatsnew/changelog.html) release notes mix past and present tense, as does [numpy](https://docs.scipy.org/doc/numpy/release.html). Not worth the trouble to create our own convention and enforce it.